src/legacy/core_plugins/interpreter/public/functions/help/font.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FontWeight } from '../../types';
+
+export const help: any = {
```

Comment:
We should always include a comment explaining why we're putting an `any` in TS.  In this case, it's because the help introspection types haven't been moved up into OSS (yet).

I would add a `TODO` comment here with a link to an issue (if one exists, otherwise create one) explaining this should be converted to use a similar structure to what exists in Canvas.

This is important.  It ensures Expressions are and remain translated.

Reply:
i moved the translations into the function as we have with the rest of the functions in OSS for now, we'll move them back when/if we add the help introspection to OSS